staging/src/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go

```diff
@@ -47,6 +49,34 @@ func init() {
 	utilfeature.DefaultMutableFeatureGate.Add(defaultKubernetesFeatureGates)
 }
 
+var (
+	CRDValidationRatcheting2 = featuregates.NewFeatureGate("CRDValidationRatcheting").
```

Comment:
Example of new creation method.  Slightly more compact and a  natural spot for versions if we want it.